### PR TITLE
qkey release v1.12

### PR DESCRIPTION
**Pull request by qiyongzhong0**
qkey release v1.12 and quick_led release v1.02

### CODE REVIEW

**Comment by supperthomas** [comment]
https://github.com/RT-Thread/packages/suites/13297090873/artifacts/725956049

**Review by mysterywolf — APPROVED** [review]

**Comment by mysterywolf** [comment]
pid的软件包分类请调整为signalprocess分类下

File: signalprocess/qpid/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "qpid",
+  "description": "A quick and easy-to-use PID automatic control algorithm package, that supports incremental and positional algorithms..",
+  "description_zh": "一款快捷易用的PID自动控制算法包，支持增量式和位置式算法。",
+  "enable": "PKG_USING_QPID",
+  "keywords": [
+    "PID"，
```
Comment (mysterywolf) [inline signalprocess/qpid/package.json:7]: 中文逗号

File: signalprocess/qpid/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "automatic control"
+  ],
+  "category": "misc",
```
Comment (mysterywolf) [inline signalprocess/qpid/package.json:10]: 分类错了

File: signalprocess/qpid/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "qpid",
+  "description": "A quick and easy-to-use PID automatic control algorithm package, that supports incremental and positional algorithms..",
```
Comment (mysterywolf) [inline signalprocess/qpid/package.json:3]:
```suggestion
  "description": "A quick and easy-to-use PID automatic control algorithm package, that supports incremental and positional algorithms.",
```

**Review by mysterywolf — CHANGES_REQUESTED** [review]

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**